core/utils/config_utils.go

```diff
@@ -0,0 +1,68 @@
+package utils
+
+import (
+	"encoding/json"
+	"errors"
+
+	"log"
+
+	"gopkg.in/yaml.v3"
+
+	"os"
+	"path/filepath"
+)
+
+const (
+	PngMimeType = "image/png"
+
+	TextRegex = `^[a-zA-Z0-9 +.,;:?!'’"“”\-_/()\[\]~&#$—%]+$`
+
+	TextCharsLimit = 500
+)
+
+var (
+	// ImageExtensions List of common image file extensions
+	// Only support PNG for now to reduce surface area of image validation
+	// We do NOT want to support formats like SVG since they can be used for javascript injection
+	// If we get pushback on only supporting png, we can support jpg, jpeg, gif, etc. later
+	ImageExtensions = []string{".png"}
+)
+
+func ReadFile(path string) ([]byte, error) {
+	b, err := os.ReadFile(filepath.Clean(path))
+	if err != nil {
+		return nil, err
+	}
+	return b, nil
+}
+
+func ReadYamlConfig(path string, o interface{}) error {
+	if _, err := os.Stat(path); errors.Is(err, os.ErrNotExist) {
+		log.Fatal("Path ", path, " does not exist")
+	}
+	b, err := ReadFile(path)
+	if err != nil {
+		return err
+	}
```

Comment:
Just let the `ReadFile` fail:
```suggestion
	b, err := ReadFile(path)
	if err != nil {
		return err
	}
```

Reply:
If you would rather decorate and log the error:
```suggestion
	if err != nil {
		log.Fatalf("Path %s: %e", path, err)
	}
```
Note if an error reading breaks our config we should crash just the same.